### DISCUSSION TITLE
使用报错，Can not be Used

### DISCUSSION THREAD

**Discussion by MacroGu** [Tangha-Technologies/UnrealEnginePluginsDemo#3] [TangHaGRPC]
![1705252041162](https://github.com/WanWanHa/MarketPlaceDemo/assets/8192020/36104933-ba8f-4814-b17b-0189ed5f6127)    

<img width="817" alt="708ab8be8506a2d1f5c01e779d6a0b4" src="https://github.com/WanWanHa/MarketPlaceDemo/assets/8192020/4f9b8196-fed5-4cf8-8226-2d4a5995ea29">   

有个这个
<img width="638" alt="96e93d15c0614d706cece207b02593b" src="https://github.com/WanWanHa/MarketPlaceDemo/assets/8192020/c48f8db9-2fc2-4736-bf96-2e1537952435">   
另外我看cmd里也只有输出到private，没有public   
![image](https://github.com/WanWanHa/MarketPlaceDemo/assets/8192020/ffc841e0-3cd8-4c8c-a9b8-bf89942af1e7)   

![1705253450704](https://github.com/WanWanHa/MarketPlaceDemo/assets/8192020/0239ab09-60cb-4dcc-bc07-a7868f80fc37)

**Comment by MacroGu** [comment]
![image](https://github.com/WanWanHa/MarketPlaceDemo/assets/8192020/4892ebf4-504a-4c50-90ad-f3e8345d1483)

**Reply by mutexShrimp** [reply]
这个插件android可以使用么？

**Comment by schoolwindows** [comment]
我遇到一个弹幕插件的问题，就是创建完端口之后，如果结束运行，再次运行，就收不到消息了！

**Reply by MacroGu** [reply]
请到 弹幕插件 下面讨论， 贴一些日志呢
https://github.com/orgs/Tangha-Technologies/discussions/categories/barrage
